Gopkg.lock

```diff
@@ -360,12 +360,61 @@
   pruneopts = "NUT"
   revision = "b3f03f55328800731ce03a164b80973014ecd455"
 
+[[projects]]
+  digest = "1:c19e61cad0c7bc59584f69c4b4c036b8271fcb48a1cdf693c5cb36efd5b537ac"
+  name = "sigs.k8s.io/kustomize"
```

Comment:
Why `kustomize` is added to this file?

Reply:
Looks like I cloned the repo in the wrong path. Let me fix this